clvm/costs.py

```diff
@@ -22,16 +22,16 @@
 DIVMOD_COST_PER_LIMB_DIVIDER = 16
 
 DIV_BASE_COST = 9
-DIV_COST_PER_LIMB_DIVIDER = 32
+DIV_COST_PER_LIMB_DIVIDER = 8
 
 SHA256_BASE_COST = 2
-SHA256_COST_PER_ARG = 1
-SHA256_COST_PER_BYTE_DIVIDER = 64
+SHA256_COST_PER_ARG = 2
+SHA256_COST_PER_BYTE_DIVIDER = 16
 
 POINT_ADD_BASE_COST = 316
-POINT_ADD_COST_PER_ARG = 4200
+POINT_ADD_COST_PER_ARG = 23000
 
-PUBKEY_BASE_COST = 4194
+PUBKEY_BASE_COST = 23000
```

Comment:
this is the major change. `point_add` and `pubkey_for_exp` become significantly more expensive.

Reply:
It seems that higher cost for `point_add` and  `pubkey_for_exp` is probably appropriate, even if it wasn't actual fact on the Raspberry Pi.

Reply:
We should probably discuss further. I know RP4 is supposed to be the "official" target, but this suggests maybe it shouldn't be. Maybe ARM isn't as well optimized as x86?

Reply:
> Maybe ARM isn't as well optimized as x86?

possibly